package.json

```diff
@@ -78,7 +78,7 @@
     "predb": "mkdir -p db",
     "db": "mongod --dbpath db",
     "db-update": "./import-tests.sh",
-    "install-openfisca": "pip install --user --upgrade -r openfisca/requirements.txt",
+    "install-openfisca": "pip install --upgrade -r openfisca/requirements.txt",
```

Comment:
`install-openfisca": "pip install --upgrade -r openfisca/requirements.txt` is not failing outside of a virtualenv but `pip install --user --upgrade -r openfisca/requirements.txt` is failing **inside** one.